custom_components/places/sensor.py

```
@@ -433,35 +450,24 @@ def extra_state_attributes(self) -> dict[str, Any]:
-    def import_persisted_attributes(self, persisted_attr: MutableMapping[str, Any]) -> None:
-        """Restore persisted runtime attributes from a stored snapshot.
+    def import_attributes_from_json(self, json_attr: MutableMapping[str, Any]) -> None:
+        """Restore persisted runtime attributes from the JSON snapshot.
 
         Args:
-            persisted_attr: Mapping loaded from persisted data.
+            json_attr: Mutable mapping loaded from the sensor's persisted JSON
+                file.
                 Imported and ignored keys are removed from this mapping.
         """
         self.set_attr(ATTR_INITIAL_UPDATE, False)
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)